scripts/deploy/localstack/buildDockerImages.sh

```diff
@@ -1,23 +0,0 @@
-# Copyright 2022-2023 Crown Copyright
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-set -e
-THIS_DIR=$(cd "$(dirname "$0")" && pwd)
-SCRIPTS_DIR=$(cd "$THIS_DIR" && cd ../.. && pwd)
-VERSION=$(cat "${SCRIPTS_DIR}/templates/version.txt")
-DOCKER_DIR="$SCRIPTS_DIR/docker"
-
-INGEST_TASK_IMAGE="sleeper-ingest-runner"
-echo "Building ingest-runner docker image"
-docker build -t "$INGEST_TASK_IMAGE" "$DOCKER_DIR/ingest"
```

Comment:
This is still needed. The Sleeper CLI doesn't currently include the ingest-runner image, just the Dockerfile and jar to build it.